src/js/init.js

```diff
@@ -2,5 +2,6 @@
     'use strict';
 
     angular.module('pmImageCrop', []);
```

Comment:
If the module was renamed maybe we can remove this line and leave just the `pmImageEditor ` @antonvas ?